### PR TITLE
Add None check on dei.xValue

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
### Changes:

- Added a None check on dei.xValue to catch invalid facts being passed back by the get_facts_dei method. This fixes a dqc_us_rules.exception:TypeError that we are seeing in Spiceweasel.

Please review: @hefischer @andrewperkins-wf 

XS-3105

### CODE REVIEW

**Comment by mindeerickson-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

File: dqc_us_rules/dqc_us_0005.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def validate_facts(val, *args, **kwargs):
     dei_list = facts.get_facts_dei(['DocumentType'], model_xbrl=val.modelXbrl)
     for dei in dei_list:
         is_an_excluded_report = any(
-            True if ex in dei.xValue else False
+            True if dei.xValue and ex in dei.xValue else False
```
Comment (jasonleinbach-wf) [inline dqc_us_rules/dqc_us_0005.py]: int 0 and "" will be falsy here.  I think you want an explicit none check.

**Comment by brianmyers-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Tested locally:
* Ran the following in wf_arelle:  pip install -U git+https://github.com/derekgengenbacher-wf/dqc_us_rules.git@XS-3105
* Tested via the testing script in wf_arelle using the two filings docs showing in the error.  Without Derek's fix, the errors were seen in the output (Doc1NoFix.txt and Doc2NoFix.txt).  With the fix, the errors no longer occurred (Doc1WithFix.txt and Doc2WithFix.txt).  

QA +1

**Comment by jasonleinbach-wf** [comment]
Reviewed and verified.  Merging.

**Pull request merged**